tox.ini

```diff
@@ -7,6 +7,7 @@ deps =
     -rrequirements.txt
     pyftpdlib
     parameterized
+    pdfminer < 20191010
```

Comment:
What happened on October 10 that broke our code?

Reply:
They went Python 3 only (sounds like a great idea....).

Reply:
Thank you for clarifying!